app/mixins/grades_option.js

```diff
@@ -0,0 +1,12 @@
+import Ember from "ember";
```

Comment:
To complement what I wrote above, here is a much better use of a mixin. Many controllers might require grades, and I can include it anywhere and it'll work immediatly